src/Components/SingleCategory.jsx

```diff
@@ -0,0 +1,28 @@
+import * as api from "../api";
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+import { useParams } from "react-router-dom";
+
+const SingleCategory = () => {
+  const [category, setCategory] = useState({});
```

Comment:
This changes from an object to an array - could you reuse your <Reviews /> component instead?